src/main/java/hudson/plugins/jacoco/JacocoBuildAction.java

```diff
@@ -33,9 +36,9 @@
  * @author Jonathan Fuerth
  * @author Ognjen Bubalo
  */
-public final class JacocoBuildAction extends CoverageObject<JacocoBuildAction> implements HealthReportingAction, StaplerProxy, Serializable {
+public final class JacocoBuildAction extends CoverageObject<JacocoBuildAction> implements HealthReportingAction, StaplerProxy, Serializable, RunAction2 {
```

Comment:
Good, the non-`transient` field could actually cause (non-Pipeline-related) problems as of 1.483.